cpp/search/searchresults.cpp

```diff
@@ -463,11 +463,15 @@ double Search::getPolicySurprise() const {
   return 0.0;
 }
 
-//Safe to call concurrently with search
 bool Search::getPolicySurpriseAndEntropy(double& surpriseRet, double& searchEntropyRet, double& policyEntropyRet) const {
-  if(rootNode == NULL)
+  return getPolicySurpriseAndEntropy(surpriseRet, searchEntropyRet, policyEntropyRet, rootNode);
+}
+
+//Safe to call concurrently with search
+bool Search::getPolicySurpriseAndEntropy(double& surpriseRet, double& searchEntropyRet, double& policyEntropyRet, const SearchNode* node) const {
```

Comment:
I know that this isn't the *cleanest* way to implement this API, but is this acceptable?  Or will this result in a lot of multi-threading issues?

Reply:
I think this will work, thanks!